src/api_server/src/request/mmds.rs

```diff
@@ -1,13 +1,53 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+use super::super::VmmAction;
+use micro_http::StatusCode;
 use request::{Body, Error, ParsedRequest};
+use vmm::vmm_config::mmds::MmdsConfig;
 
 pub fn parse_get_mmds() -> Result<ParsedRequest, Error> {
     Ok(ParsedRequest::GetMMDS)
 }
 
-pub fn parse_put_mmds(body: &Body) -> Result<ParsedRequest, Error> {
+pub fn parse_put_mmds(body: &Body, first_token: Option<&&str>) -> Result<ParsedRequest, Error> {
+    let mut config_request = false;
+    if let Some(config_path) = first_token {
+        if *config_path == "config" {
+            config_request = true;
```

Comment:
Hmm, why do you need the extra `config_request` bool and not include the whole logic from the next `if` block here?